cli/src/ui.rs

```diff
@@ -340,6 +340,18 @@ impl Ui {
         self.stdout().flush()?;
         let mut buf = String::new();
         io::stdin().read_line(&mut buf)?;
+
+        if buf.ends_with('\n') {
+            let mut chars = buf.chars();
+            chars.next_back();
+            buf = chars.as_str().to_owned();
+        } else if buf.is_empty() {
+            return Err(io::Error::new(
+                io::ErrorKind::UnexpectedEof,
+                "Prompt cancelled by EOF",
+            ));
+        }
```

Comment:
I think this can be simplified by using [strip_suffix](https://doc.rust-lang.org/std/primitive.str.html#method.strip_suffix)

Reply:
Done.